worker/src/initialize.ts

```diff
@@ -0,0 +1,3 @@
+import { upsertDefaultModelPrices } from "./scripts/upsertDefaultModelPrices";
+
+upsertDefaultModelPrices();
```

Comment:
is it intended that this here is not awaiting?

Reply:
yes, we are running this at server bootstrap and there is no top-level await in node currently. This is in any case fine to run async 👍🏾